pkg/tfsandbox/plan.go

```diff
@@ -73,5 +73,11 @@ func (t *Tofu) planWithOptions(ctx context.Context, logger Logger, refreshOnly b
 		return nil, fmt.Errorf("error running show plan: %w", err)
 	}
 
+	humanPlan, err := t.tf.ShowPlanFileRaw(ctx, planFile, tfexec.JSONNumber(true))
```

Comment:
Can probably run in parallel with ShowPlanFile right?

Reply:
Updated!